src/components/Root.tsx

```diff
@@ -0,0 +1,8 @@
+import { classnames, margin, padding } from 'classnames/tailwind'
+import ChildrenProp from 'models/ChildrenProp'
+
+const root = classnames(margin('mx-auto'), padding('pb-10'))
```

Comment:
`pb-10` is for `Footer`? I think we shouldn't do this, maybe we can do something inside the `Footer` itself?

Reply:
Why not? It's a root wrapper, and Footer is `absolute`. Small padding at the end of the page is not a bad idea as I think

Reply:
Let's update the `Footer` then:
```
.grid-wrapper {
  width: 100vw;
  height: 50vh;
  position: relative;
  perspective: 180px;
  perspective-origin: 50% 20%;
  overflow: hidden;
}

```

Reply:
Changed it. Everything works fine. Thank you